src/app/admin/admin-reports/filtered-items/filtered-items.component.html

```diff
@@ -0,0 +1,175 @@
+<div class="container">
+    <div class="metadata-registry row">
+        <div class="col-12">
+
+            <h2 id="header" class="border-bottom pb-2">{{'admin.reports.items.head' | translate}}</h2>
```

Comment:
Similar to above, this should be an `<h1>` for accessibility

Reply:
Done.